### ISSUE TITLE
Marquee direction for scrollers, continuous scrolling feature

### ISSUE THREAD

**Issue by turgo** [spdawson/lcdapi#4]
I missed the continuous scrolling feature, so I modified your source:

```
diff --git a/lcdapi/api/LCDWidget.h b/lcdapi/api/LCDWidget.h
index db97d8f..e2087ac 100644
--- a/lcdapi/api/LCDWidget.h
+++ b/lcdapi/api/LCDWidget.h
@@ -52,7 +52,8 @@ class LCDWidget : public LCDElement
   enum Direction
   {
     Horizontal = 'h',
-    Vertical = 'v'
+    Vertical = 'v',
+    Marquee = 'm'
   };
   /**
    * \brief A list of parameters for set method.
```

I think something like that should be implemented. Thanks in advance!

**Comment by spdawson** [comment]
Thanks for this; released v0.10 with support for `Marquee`.

**Comment by turgo** [comment]
Thanks!
